addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -221,29 +228,15 @@ var ImageWidget = MediaWidget.extend({
 
         this.multiImages = options.multiImages;
 
-        this.firstFilters = options.firstFilters || [];
-        this.lastFilters = options.lastFilters || [];
```

Comment:
@sdegueldre In fact, instead of removing variables stored in `this`, can you leave them and add a comment "remove in ..." ... choose what you prefer (master, saas-13.3, whatever). So that potential custo relying on this.XXX don't crash, just stop working.